packages/lolcat.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Lolcat < Package
+  description 'Rainbows and unicorns!'
+  homepage 'https://github.com/busyloop/lolcat'
+  version '99.9.19'
+  source_url 'https://github.com/busyloop/lolcat/archive/v99.9.19.tar.gz'
+  source_sha256 '2ec5beb8051576600adf3197b9826ce3319a18448bd0dbb8a92a028be8370449'
+
+  def self.install
+    system "gem install lolcat"
```

Comment:
Probably want to install in `#{CREW_DEST_PREFIX}` somehow, otherwise, files aren't tracked and nothing shows up with `crew files lolcat`.

Reply:
Would `gem install lolcat --install-dir #{CREW_DEST_LIB_PREFIX}/ruby/gems/*` work?

Reply:
You know what, I think trying to make packages of gem packages just isn't worth it. Just let them use gem 😂
Perhaps someday we could add stuff to the `Package` class that would give us the option to "metapackage" from other package managers... not really worth it yet, though.